Fix missing Refinery helpers in PagesController

rottame opened the pull request:
```
NameError in Pages#home

Showing /home/ngl/.rvm/gems/ruby-1.9.2-p136/gems/refinerycms-core- 0.9.9.13/app/views/shared/_content_page.html.erb where line #8 raised:

undefined local variable or method `page_title' for #<#<Class:0x944b444>:0x9446a84>
```

Wrapping the monkeypatch for PagesController in refinery.on_attach instead of config.to_prepare in lib/refinerycms_memberships.rb fixes the problem for me.

There is a problem: in refinerycms master, the callback was renamed to refinery.after_inclusion

Commit:
Fix missing Refinery helpers in PagesController

Commit:
Grant access on reserved pages to admins

rbriank commented:
Thanks!  I was talking to Phil about this last night but didn't get a chance
to do it.  I'll pull your changes in, hopefully, tonight.

B

On Wed, Mar 30, 2011 at 8:35 AM, rottame <
reply@reply.github.com>wrote:

> NameError in Pages#home
> 
>    Showing /home/ngl/.rvm/gems/ruby-1.9.2-p136/gems/refinerycms-core-
> 0.9.9.13/app/views/shared/_content_page.html.erb where line #8 raised:
> 
>    undefined local variable or method `page_title' for
> #<#Class:0x944b444:0x9446a84>
> 
> Wrapping the monkeypatch for PagesController in refinery.on_attach instead
> of config.to_prepare in lib/refinerycms_memberships.rb fixes the problem for
> me.
> 
> There is a problem: in refinerycms master, the callback was renamed to
> refinery.after_inclusion
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/rbriank/refinerycms_membership/pull/4

Commit:
Redirect unauthenticated user to members#login

Commit:
Load page record
Why show the members list to other members? redirect to login

Commit:
Seeds (pages, roles)

Commit:
Login action

Commit:
Redirect to the page that triggered the login if not admin

Commit:
Don't show membership level on create

Commit:
Use partials
More refinery like rendering (use shared/content_page)
Render part :body_above before the form
Render part :body_below after the form

Commit:
Why index is a list of members? Remove

Commit:
Make memberships a singleton controller, remove administration from front end

Commit:
Custom warden failure class to redirect to /members/login on login failure

Commit:
More pages

Commit:
Hide extra parts from content_page, better form

Commit:
Do not create a useless form

Commit:
Use shared/content_page to render the view, wrap the form between :page_above and :page_below

Commit:
Profile page, wrapped between :page_above and :page_below

Commit:
Thank you page after registration

Commit:
Remove show action from sigleton controller

Commit:
Use recall instead of redirect on login error

Commit:
Login flash messages

Commit:
Layout fixes, I18n

Commit:
Add role management tab to page editor

Commit:
Remove pages roles tab from memberships page

Commit:
Roles administration

Commit:
Start administration pages work

Commit:
Delete unused files

Commit:
New admin page structure, Stub members controller, Roles controller, New grid

Commit:
Fix wrong links on members views

Commit:
Fix javascript error on page without parameters

Commit:
Fix redirect in some cases

Commit:
More work on user management, activation fields in user table, admin/members#edit action

Commit:
Members administration almost done

Commit:
Begin email administration

Commit:
Menu icons, shared partial

Commit:
:text means nothing, rename to :body; do not create admin emails...

Commit:
Email editors, admin email logic removed

Commit:
Email settings

Commit:
it translation, strings

Commit:
Duplicate icon removed

Commit:
String fix

Commit:
Common email parts administration

rottame committed:
Email body help stub, strings updated

rbriank commented:
I'd like to add you to my professional network on LinkedIn.
- Brian

Brian  Kierstead
Information Technology and Services Professional
Ontario, Canada

Confirm that you know Brian  Kierstead:
https://www.linkedin.com/e/-cff684-gui8ozav-2s/isd/4773951230/043mHC7G/?hs=false&tok=0JdvEK2wOIM4Y1

## 

You are receiving Invitation to Connect emails. Click to unsubscribe:
http://www.linkedin.com/e/-cff684-gui8ozav-2s/vEXs_3Sofs3M3LUCKykvylxR03SESUaKngFDJD0Rf3jEUapMV_F-_cj9cM0Voat28gOFVc5xN_0EZdaiq_HBiN19dRyKYaL1mF5aP9/goo/reply%2Bi-717569-1590d399f260364a93325da009ce45d2fa858acb%40reply%2Egithub%2Ecom/20061/I1659926204_1/?hs=false&tok=14IxuAnIGIM4Y1

(c) 2011 LinkedIn Corporation. 2029 Stierlin Ct, Mountain View, CA 94043, USA.

Commit:
Fix for Refinery 0.9.9.14+

Commit:
Timestamp migration in correct order

Commit:
Plugin title in italian

Commit:
Initialize settings

Commit:
Settings refactoring

Commit:
Image dialog and WYMEditor patches: save image id and resizing options as data attributes of the IMG tag

Commit:
More work on registration, email, user confirmation by email

Commit:
Rework grid javascript

Commit:
Fix Admin::UsersController monkeypatch

Commit:
Members filters

Commit:
Settings fix

Commit:
Actions fix

Commit:
Actions fix

Commit:
Status fix

Commit:
Email body help

Commit:
Status fix

Commit:
Reload parent page on settings saved

Commit:
Various fixes

Commit:
Fix test of User class

Commit:
Deny access to admin pages to members

Commit:
Make model modular, ease model override

Commit:
Fix validations, save

Commit:
Fix

Commit:
Fixes

Commit:
fixes

Commit:
Validations, protect ApplicationController, Fix images

Commit:
Sender address

Commit:
Preferences

Commit:
Reset password

Commit:
show

Commit:
fix
